src/layout/barGrid.js

```diff
@@ -478,7 +478,9 @@ export function layout(seriesType, ecModel) {
                 x: x,
                 y: y,
                 width: width,
-                height: height
+                height: height,
+                r: seriesModel.get('barBorderRadius'),
```

Comment:
I believe this should be `itemModel.get('barBorderRadius')`?

Reply:
@pissang item border radius wasn't set from here. so I have it removed.